lib/src/strings.dart

```diff
@@ -29,6 +29,19 @@ String get dynamicLibParentName => Platform.isWindows ? 'bin' : 'lib';
 
 const output = 'output';
 
+const language = 'language';
+
+// String mappings for the Language enum.
+const langC = 'c';
+const langObjC = 'objc';
+
+// Clang command line args for Objective C.
+const clangLangObjC = ['-x', 'objective-c'];
```

Comment:
Should we also pass a language flag when lang is `C`? Not sure if that would make any difference though.

Reply:
It defaults to C, so this isn't needed.